docs/config.md

```diff
@@ -0,0 +1,168 @@
+Example:
+```
+grpc:
+  listen_address: "localhost:8282"
+  connection_timeout_seconds: 160 
+  certs:
+    tls_cert_path: "/app/grpc.crt"
+    tls_key_path: "/app/grpc.key"
+    tls_ca_cert_path: "/app/grpc-ca.crt"
+```
+
+#### 2. gateway
+The gateway section allows configuring the [grpc gateway](https://github.com/grpc-ecosystem/grpc-gateway) for your topaz authorizer. 
+
+- *listen_address* - string - allows the topaz Gateway server to spin up on the requested port (default: "0.0.0.0:8383")
+- *http* - boolean - when set to true it allows the gateway service to respond to plain http request (default: false)
+- *certs* - certs.TLSCredsConfig - based on [aserto-dev/certs](https://github.com/aserto-dev/certs) package allows setting the paths of your certificate files. If you do not have your certificates in the specified paths, topaz will generate self-signed certificates for you. By default topaz will generate the certificates in ` ~/.config/topaz/certs/` path
+- *allowed_origins* - []string - allows setting the paths for the [CORS handler](https://github.com/rs/cors)
+
+Detailed information about the gateway http server timeout configuration is available [here](https://pkg.go.dev/net/http#Server)
+- *read_timeout* - time.Duration - default value set to 2 * time.Second (default: 2000000000) 
+- *read_header_timeout* - time.Duration - default value set to 2 * time.Second (default: 2000000000) 
+- *write_timeout* - time.Duration - default value set to 2 * time.Second (default: 2000000000) 
+- *idle_timeout* - time.Duration - default is set to 30 * time.Second (default: 30000000000)
+
+Example:
+```
+gateway:
+  listen_address: "localhost:8383"
+  http: false
+  certs:
+    tls_cert_path: "/app/gateway.crt"
+    tls_key_path: "/app/gateway.key"
+    tls_ca_cert_path: "/app/gateway-ca.crt"
+  allowed_origins:
+  - https://*.aserto.com
+  - https://*aserto-console.netlify.app
+  - https://*aserto-playground.netlify.app
+```
+
+#### 3. health
+The health is the simplest of these section as it only allows to set the listen address of the health server. By default the listen address is set to 0.0.0.0:8484
+
+To check the health of your service you can use the available [grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe) 
+Example:
+```
+grpc-health-probe --addr=localhost:8484
+status: SERVING
+```
+
+### c. Directory Service
+
+Topaz is able to communicate with a directory service based on the [pb-directory proto](https://github.com/aserto-dev/pb-directory) definitions. When the remote address is configured to localhost, topaz is able to spin-up a grpc [edge directory service](https://github.com/aserto-dev/go-edge-ds) based on [bbolt](https://pkg.go.dev/go.etcd.io/bbolt)
+
+Example:
+```
+directory_service:
+  edge:
+    db_path: /app/dbs/eds-acmecorp.db
+    seed_metadata: false
+  remote:
+    address: "localhost:12345"
+    insecure: true
+```
+
```

Comment:
perhaps add an example for configuring a remote directory (not localhost?) - including apikey, tenant id headers...